nomic/chain.json

```diff
@@ -71,14 +71,9 @@
         "provider": "Golden Ratio Staking"
       },
       {
-        "id": "10beadbcd4bc5fef8a1f5f57353bdb8646f7a554",
-        "address": "seeds.whispernode.com:26656",
-        "provider": "WhisperNode🤐"
-      },
-      {
-        "id": "27b6d74c8408e033e2e5a9e966a0d15782e33596",
-        "address": "seeds.nethernode.xyz:12756",
-        "provider": "carbonZERO🌲"
```

Comment:
Not sure why CarbonZero nodes are being removed?

Reply:
team is no longer providing support / validating for the chains I cleaned up (they're friends of mine and was just helping them out). can have them submit this change if necessary, but they only validate AssetMantle / Nomic / Stride (governor) at this point, with no additional support being provided currently.